### PR TITLE
[DOCFIX] Update Chinese Docker Doc

### PR DESCRIPTION
**Pull request by thanosaw**
Fixed issues with past PR

### CODE REVIEW

File: docs/cn/deploy/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -1,232 +1,318 @@
...
+-e ALLUXIO_JAVA_OPTS="-Dalluxio.property.name=value"
+```
 
-通过 `--build-arg`，将本地或远程的Alluxio压缩包搭建Alluxio Docker镜像
+多个属性应以空格分隔。
 
-本地压缩包:
-```console
-$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+如果属性值包含空格，则必须使用单引号将其转义。
```
Comment (LuQQiu) [inline docs/cn/deploy/Running-Alluxio-On-Docker.md:213]: will it be better to change to `使用单引号来将带空格的字符串考虑为一个值`？

File: docs/cn/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -1,232 +1,318 @@
...
+  --device /dev/fuse \
+  alluxio-fuse fuse \
 ```
-FROM ubuntu:16.04
-```
 
-然后你可以用和创建支持FUSE的镜像一样的命令来创建镜像并且运行它。[这里](https://hub.docker.com/r/alluxio/alluxio-tensorflow/)有一个预先创建好的带有TersorFlow的docker镜像。
+## 故障排除
+
+可以通过运行docker访问Alluxio服务器日志 logs $ container_id。通常，日志可以很好地指出什么地方出了问题。如果它们不足以诊断您的问题，则可以在[用户邮件列表](https://groups.google.com/forum/#!forum/alluxio-users).上获得帮助 。
```
Comment (LuQQiu) [inline docs/cn/deploy/Running-Alluxio-On-Docker.md:318]: 可以通过运行 `docker logs $container_id` 来访问Alluxio 服务器日志。
the docker logs $ contianer_id is a single command with backticks, so we can keep them the same when translating other texts

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**